lib/irc/ConnectionInstance.js

```diff
@@ -19,7 +19,9 @@ const PING_TIMEOUT_MS = 1000 * 60 * 10;
 const THROTTLE_WAIT_MS = 20 * 1000;
 
 // The minimum time to wait between connection attempts if the kernel has given us
-// a net_error.
+// a net_error. Usually these come about due to a network being unavaliable or resource
+// starvation. Unlike throttling which is ircd based, we should retry sooner to see if
+// the kernel is happy now.
```

Comment:
But why? :innocent:

Reply:
Can I not just say "please consult your kernel's manual"?

Reply:
If we can justify backing off more than the base ~2s, then let's do it. Otherwise, I'm not sure there's a problem.